.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
 name: Continuous Integration
 
 on:
-  # pull_request:
-  #   branches: ["*"]
+  pull_request:
```

Comment:
**thought:** this meant PRs wouldn't have any CI whatsoever, seems to be by omission.